source/internal/_nth.js

```diff
@@ -0,0 +1,30 @@
+import _isString from './_isString.js';
+
+/**
```

Comment:
unless you feel there is something worth mentioning here, we don't need these docs in the internal definitions, I'd say remove them

Reply:
You're right, the idea was that since I moved the entire logic to internal, I also copied the docs with it, but it is not necessary, so I removed it.